ci/kind/kind-upgrade.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+LATEST_KIND_VERSION=$(curl -sSL https://api.github.com/repos/kubernetes-sigs/kind/releases/latest | jq -r '.tag_name' | awk '{print substr($1, 2)}')
+
+# on linux
+function upgrade_kind {
+    runtimeOS="$(uname -m)"
+    binary_name=""
+    case "$runtimeOS" in
+     "x86_64")
+      binary_name="kind-linux-amd64"
+      ;;
+      "aarch64")
+      binary_name="kind-linux-arm64"
+    esac
+    curl -Lo ./kind https://kind.sigs.k8s.io/dl/$LATEST_KIND_VERSION/$binary_name
+    chmod +x ./kind
+    sudo mv ./kind /usr/local/bin/kind
```

Comment:
Wouldn't this create an issue if there is already an existing  installation of kind.

Reply:
I think it will be override during upgradation.

Reply:
Is it verified?, have you tried running this script?

Reply:
yes, on Linux vm.